File: L1/1.2/app/src/main/java/com/example/gradoktorazawadyjava/MainActivity.java
package com.example.gradoktorazawadyjava;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private int r1 = 0;
    private int r2 = 0;
    private int score = 0;

    private TextView points;
    private Button left;
    private Button right;
    private Random random;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        left = findViewById(R.id.leftButton);
        right = findViewById(R.id.rightButton);
        points = findViewById(R.id.points);
        score = 0;
        random = new Random();
        roll();
    }

    private void roll(){
        points.setText("Points: " + score);
        r1 = random.nextInt(16);
        r2 = random.nextInt(16);
        left.setText(Integer.toString(r1));
        right.setText(Integer.toString(r2));
    }

    public void leftClick(View view) {
        if(r1 >= r2){
            score++;
            Toast.makeText(this, "Dobrze!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }

    public void rightClick(View view) {
        if(r1 <= r2){
            score++;
            Toast.makeText(this, "Dobrze!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }
}

File: L1/1.1/app/src/main/java/com/example/gradoktorazawadykotlin/MainActivity.kt
package com.example.gradoktorazawadykotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private var r1 = 0
    private var r2 = 0
    private var score = 0


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        score = 0
        roll()
    }

    fun roll() {
        findViewById<TextView>(R.id.points).text = "Punkty: " + score
        r1 = Random.nextInt(16)
        r2 = Random.nextInt(16)
        findViewById<TextView>(R.id.leftButton).text = "" + r1
        findViewById<TextView>(R.id.rightButon).text = "" + r2
    }

    fun rightClick(view: View) {
        if (r1<=r2){
            score++
            Toast.makeText(this, "Dobrze!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
    fun leftClick(view: View) {
        if (r1>=r2){
            score++
            Toast.makeText(this, "Dobrze!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
}